fabric-model-events-api-v1/src/client/java/net/fabricmc/fabric/mixin/client/modelevents/ModelPartMixin.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.client.modelevents;
+
+import java.util.List;
+import java.util.Map;
+
+import org.jetbrains.annotations.ApiStatus;
+import org.spongepowered.asm.mixin.Dynamic;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.gen.Accessor;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.fabricmc.fabric.impl.client.modelevents.FabricPartHooks;
+import net.minecraft.client.model.ModelPart;
+import net.minecraft.client.model.ModelPart.Cuboid;
+import net.minecraft.client.render.VertexConsumer;
+import net.minecraft.client.util.math.MatrixStack;
+
+/**
+ * RE: priority
+ *     This mixin is applied last to ensure any other mods that want to inject are already applied before us.
+ *     This is to prevent our event being overwritten by (e.g. sodium) which may be making significant changes
+ *     to the render() method
+ */
+@ApiStatus.Internal
+@Mixin(value = ModelPart.class, priority = Integer.MAX_VALUE)
+abstract class ModelPartMixin implements FabricPartHooks.Container {
+    @Shadow
+    public boolean visible;
+    @Shadow
+    public boolean hidden;
+
+    private FabricPartHooks fabric_hooks;
+
+    @Override
+    @Accessor("cuboids")
```

Comment:
Yeah. It works.

Reply:
To offer a better explanation:
This works because I specified the field name on the @Accessor annotation instead of letting try to inspect the method name to find one. That's leaves the method name open for me to name it anything I want, in this case to implement the interface.